src/main/scala/devsearch/features/FeatureRecognizer.scala

```diff
@@ -100,19 +106,19 @@ case class CodePiecePosition(location: CodeFileLocation, line: Int) extends java
   override def toString = location.toString + ":" + line
 }
 
-case class CodeFile(language: String, location: CodeFileLocation, ast: AST) extends java.io.Serializable
+case class CodeFile(location: CodeFileLocation, ast: AST) extends java.io.Serializable
```

Comment:
Why are you removing the language? This could potentially be used during feature extraction. We don't do it currently, but only because we're lazy and short on time ;)